include/exadg/grid/mapping_dof_vector.h

```diff
@@ -145,7 +145,7 @@ class MappingDoFVector : public dealii::MappingQCache<dim>
 
     for(auto const & cell : dof_handler.active_cell_iterators())
     {
-      if(!cell->is_artificial())
+      if(not cell->is_artificial())
```

Comment:
`not(...)`?

Reply:
The problem is that introducing brackets here would be hard to realize in an automatic manner by search-and-replace scripts (I did not know how to do this).